### DISCUSSION TITLE
How to create report file as "SpotBugs Bug Detector Report" ?

### DISCUSSION THREAD

**Discussion by TsingboWoo** [spotbugs/spotbugs-maven-plugin#917] [Q&A]
How to create report file as "SpotBugs Bug Detector Report" ?

I use the latest version of Spotbugs-maven-plugin 4.8.6.5 and SpotBugs 4.8.6，
set the htmlOutput to true，the HTML report is only the default format of default.xsl,  SpotBugs Report.
                    <xmlOutput>true</xmlOutput>
                    <htmlOutput>true</htmlOutput>

How can I to make it to the same as the following example：
https://spotbugs.github.io/spotbugs-maven-plugin/examples/spotbugs.html

**Comment by hazendaz** [comment]
mvn site generates that, is that what you mean?
